source/euler/hyperbolic_system.h

```diff
@@ -1313,5 +1339,159 @@ namespace ryujin
       return result;
     }
 
+
+    template<int dim, typename Number>
+    void HyperbolicSystem::View<dim, Number>::post_process(const vector_type &U,
+                                                           Number t,
+                                                           MPI_Comm mpi_communicator_,
+                                                           const ryujin::OfflineData<dim, Number>& offline_data_)
+    const
+    {
+    if(dim == 2){
+      using scalar_type = dealii::LinearAlgebra::distributed::Vector<Number>;
+
+
+       //some general variables
+       dealii::Point<dim> disk_center;//center at 0,0 for every case, how to not hardcode this?
+       //FIXME: how do we calculate disk center from the information in mesh generation?
+       double disk_radius = 0.25;
```

Comment:
This is unnecessarily difficult to read because you don't use a consistent indentation. Make sure nested statements are indented (the `if` above isn't) and that things are *consistently* indented (the `using` is, but things below are not aligned).

Reply:
Perhaps applying the style file to the whole project will fix this, But I can also fix this. Did not intend for this to be difficult to read. Slightly annoying that this only works for dimension 2. Maybe there is a way I can generalize the code to work in higher dimensions. 

The main thing I need to get around is changing the tensor describing the forces into a 3d tensor and describing the drag as some sort of scalar product with the direction of motion.